### DISCUSSION TITLE
Browser history is not working on the Github site

### DISCUSSION THREAD

**Discussion by ziroc** [community/community#76890] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Basically the title. For example I am browsing in a given tab and at some point I load my github repo. Then the browser history for this tab gets deleted. I can only see new tab > github repository page. 
If I move between pages, for example to Issues then Pull Requests, again the history is not working. If I click back I will go to the new tab page.
I am using Firefox with a few addons, so I'm not sure if it's something on my side. Have somebody else experienced this?

**Comment by curiosity-a** [comment]
I am able to go back to pre-github pages, so that is probably one of your (privacy?) addons misbehaving.

But the navigation over the site itself is an abysmal experience. Not only does it work intermittently, but when it does, it is also extremely slow compared to the browser's builtin navigation. The only saving grace is that the page URL changes, so you can refresh the page when the navigation inevitably breaks.

I wish I could opt out of this navigation override and go back to using fast and reliable navigation the browser itself provides.

**Comment by ziroc** [comment]
Alright. I found the problem. 
I am using this setting in Firefox: "browser.navigation.requireUserInteraction = true"
I use it because I don't like some sites messing with the browser history. 
It seems though that Github needs this for some reason. Which sounds to me like a hack or bad coding.
